core/spec/models/spree/order/finalizing_spec.rb

```diff
@@ -48,19 +48,25 @@
     end
 
     it "should send an order confirmation email" do
+      observer = Spree::EmailObserver.new
+      order.add_observer(observer)
```

Comment:
This is the thing where maybe these tests can just be that the observers get notified, so we can just make a mock one and add it. 

Alternatively, if you want to keep the tests as they are, you could add this to the `before` code, and then you don't need to have this consistently doing it, but it'll always be added.